recipe/meta.yaml

```
@@ -15,21 +23,35 @@ source:
     - 0001-windows-find-blas.patch
 
 build:
-  script: CMAKE_GENERATOR="Ninja" {{ PYTHON }} -m pip install . -vv  # [not win]
+  script:
+    # cuQuantum only supports Volta+
+    # TODO: add cc90 here when we migrate to CUDA 11.8/12.0
+    - export AER_CUDA_ARCH="7.0;7.5;8.0;8.6+PTX"  # [linux64 and cuda_compiler_version != "None"]
+    - CMAKE_GENERATOR="Ninja" {{ PYTHON }} -m pip install . -vv  # [not win]
   # Skip macOS build until https://github.com/Qiskit/qiskit-aer/issues/1689 is addressed
-  skip: true  # [osx]
-  number: 1
+  skip: true  # [osx or (win64 and cuda_compiler_version != "None")]
+  number: {{ build }}
+  string: cuda{{ cuda_compiler_version | replace('.', '') }}_py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [cuda_compiler_version != "None"]
+  string: cpu_py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}                                                 # [cuda_compiler_version == "None"]
   script_env:
     - DISABLE_DEPENDENCY_INSTALL=ON
     - DISABLE_CONAN=ON
     - CMAKE_GENERATOR=Ninja
     - CMAKE_PREFIX_PATH=%PREFIX%  # [win]
+    - CMAKE_BUILD_PARALLEL_LEVEL=1  # [linux64 and cuda_compiler_version != "None"]
+    - CUQUANTUM_ROOT=$PREFIX  # [linux64 and cuda_compiler_version != "None"]
+    - CUTENSOR_ROOT=$PREFIX  # [linux64 and cuda_compiler_version != "None"]
+    - AER_THRUST_BACKEND=CUDA  # [linux64 and cuda_compiler_version != "None"]
+    - AER_ENABLE_CUQUANTUM=ON  # [linux64 and cuda_compiler_version != "None"]
+  ignore_run_exports_from:
+    - {{ compiler('cuda') }}  # [linux64 and cuda_compiler_version != "None"]
 
 requirements:
   build:
     - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}  # [linux64 and cuda_compiler_version != "None"]
```

Think we can simplify to...

```suggestion
    - {{ compiler('cuda') }}  # [cuda_compiler != "None"]
```

---

Addressed in #21.